receiver/sqlserverreceiver/queries.go

```diff
@@ -0,0 +1,64 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sqlserverreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlserverreceiver"
+
+import (
+	"fmt"
+)
+
+// Direct access to queries is not recommended: The receiver allows filtering based on
+// instance name, which means the query will change based on configuration.
+// Please use getSQLServerDatabaseIOQuery
+const sqlServerDatabaseIOQuery = `
+SET DEADLOCK_PRIORITY -10;
+IF SERVERPROPERTY('EngineEdition') NOT IN (2,3,4) BEGIN /*NOT IN Standard,Enterprise,Express*/
+	DECLARE @ErrorMessage AS nvarchar(500) = 'Connection string Server:'+ @@ServerName + ',Database:' + DB_NAME() +' is not a SQL Server Standard,Enterprise or Express. This query is only supported on these editions.';
+	RAISERROR (@ErrorMessage,11,1)
+	RETURN
+END
+
+DECLARE
+	 @SqlStatement AS nvarchar(max)
+	,@MajorMinorVersion AS int = CAST(PARSENAME(CAST(SERVERPROPERTY('ProductVersion') AS nvarchar),4) AS int) * 100 + CAST(PARSENAME(CAST(SERVERPROPERTY('ProductVersion') AS nvarchar),3) AS int)
+	,@Columns AS nvarchar(max) = ''
+	,@Tables AS nvarchar(max) = ''
+IF @MajorMinorVersion > 1100 BEGIN
+	SET @Columns += N'
+	,vfs.[io_stall_queued_read_ms] AS [rg_read_stall_ms]
+	,vfs.[io_stall_queued_write_ms] AS [rg_write_stall_ms]'
+END
+
+SET @SqlStatement = N'
+SELECT
+	''sqlserver_database_io'' AS [measurement]
+	,REPLACE(@@SERVERNAME,''\'','':'') AS [sql_instance]
```

Comment:
```suggestion
	,@@SERVERNAME AS [sql_instance]
```
I lean towards no formatting the instance name. If the user wants to filter based on the instance name returning a different instance name than what they wanted to filter would likely cause confusion.